lib/std/debug.zig

```diff
@@ -20,7 +20,7 @@ const windows = std.os.windows;
 pub const leb = @import("debug/leb128.zig");
 
 pub const FailingAllocator = @import("debug/failing_allocator.zig").FailingAllocator;
-pub const failing_allocator = &FailingAllocator.init(global_allocator, 0).allocator;
+pub const failing_allocator = &FailingAllocator.init(&global_fixed_allocator.allocator, 0).allocator;
```

Comment:
Maybe this should also be moved into `std.testing`?

Reply:
:+1: